### PR TITLE
[FEAT]: Added Cart page

### PR DESCRIPTION
**Pull request by fahd23**
Added Cart page

Screenshot (83)

### CODE REVIEW

**Review by mariyasada — APPROVED** [review]
looks good. you can merge it

File: cart/cart.html [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+          <div class="card-body">
+            <div>
+              <p class="disc-price">₹850</p>
+              <p class="actual-price">₹1700</p>
+              <p class="offer-info">(50% OFF)</p>
+            </div>
+            <div>
+              <h3>
+                Quantity:
+                <input type="number" id="quantity-input" value="1" min="1" />
```
Comment (Avin008) [inline cart/cart.html]: why use id here?

File: cart/cart.css [hunk 2]
```diff
@@ -0,0 +1,45 @@
+@import url(/main.css);
+@import url(/responsive.css);
+
+hr {
+  border: 0;
+  border-bottom: 1px solid var(--black);
+}
+#quantity-input {
```
Comment (Avin008) [inline cart/cart.css]: using id as a selector. you can switch to class for consistency

**Review by Avin008 — APPROVED** [review]
Everything looks great. good to merge

File: cart/cart.html (hunk 1 above)
Comment (fahd23) [reply cart/cart.html]: No particular reason.

File: cart/cart.css (hunk 2 above)
Comment (fahd23) [reply cart/cart.css]: Ok

**Pull request merged**